### PR TITLE
Fwdport/6.2.5 6.3

### PR DESCRIPTION
**Pull request by rahulreddy**

### CODE REVIEW

**Comment by rahulreddy** [comment]
@ironman-machine try

**Comment by rahulreddy** [comment]
End to passed here http://ci.ironmann.io/gh/scality/Integration/8996#build-parameters

File: lib/routes.js [hunk 1]
```diff
@@ -100,5 +100,12 @@ export default function routes(req, res, logger) {
             undefined, res, log);
     }
 
+    if ((req.query.partNumber || req.query.uploadId)
```
Comment (bennettbuchanan) [inline lib/routes.js]: Part numbers can be any number from 1 to 10,000, inclusive.
http://docs.aws.amazon.com/AmazonS3/latest/API/mpUploadUploadPart.html

File: lib/routes/routePOST.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import api from '../api/api';
 import routesUtils from './routesUtils';
 import pushMetrics from '../utilities/pushMetrics';
 
+const MAX_POST_LENGTH = 1024 * 1024; // 1 MB
```
Comment (bennettbuchanan) [inline lib/routes/routePOST.js]: It was. Updated in https://github.com/scality/S3/pull/567/commits/0d52ada141e7743b76e80347b3a2c5c6908acd43.

Comment (ploki) [inline lib/routes/routePOST.js]: ok

File: lib/routes.js (hunk 1 above)
Comment (ploki) [inline lib/routes.js]: ok

**Comment by rahulreddy** [comment]
Please don't merge this yet. We are waiting for one more PR.

**Comment by rahulreddy** [comment]
Triggered end to end here http://ci.ironmann.io/gh/scality/Integration/9001#build-parameters

**Comment by rahulreddy** [comment]
End to end re-triggered here http://ci.ironmann.io/gh/scality/Integration/9003#build-parameters

**Comment by rahulreddy** [comment]
End to end passed

**Pull request merged**